test/dlc_tests/ec2/test_efa.py

```diff
@@ -35,36 +35,6 @@
 )
 
 
-@pytest.mark.processor("gpu")
```

Comment:
this came due to rebase I think, deleting it as its just empty func and not needed